Sources/SpeziConsent/Resources/ConsentTest.md

```diff
@@ -0,0 +1,32 @@
+---
+study: MHC
+title: MyHeart Counts
+---
+
+# Welcome to MyHeart Counts
+
+Heyyy
+
+- spezi
+- dot
+- stanford
+- dot
+- edu
+
+how about some numbers?
+
+1. one
+2. two
+3. three
+
+<toggle id=toggle1 default=false>Are you sure you have understood the instructions?</toggle>
+
+<select id=select1 default=de>
+    <option id=de>I am German</>
+    <option id=us>I am American</>
+    <option id=uk>I am British</>
+</select>
```

Comment:
Might make sense to add a test with a non-default value? Generally would be the more common use case; in general we won't have a lot of default values in a consent document.